std_msgs/Duration duration

MotorRequest motor1
MotorRequest motor2
MotorRequest motor3
MotorRequest motor4

ServoRequest servo1
ServoRequest servo2
ServoRequest servo3
ServoRequest servo4

ServoRequest pan
ServoRequest tilt

std_msgs/String moveUuid
